### PR TITLE
Move crc under code-ready directory

### PR DESCRIPTION
**Pull request by praveenkumar**
It would be helpful so other project under code-ready org should be
listed under code-ready directory.

### CODE REVIEW

**Comment by gbraad** [comment]
> /restest

that won't work...

however, perhaps the name should be considered? just a general thought ...

File: ci-operator/config/code-ready/crc/code-ready-crc-master.yaml [hunk 1]
```diff
@@ -35,7 +35,7 @@ tests:
   as: e2e-crc
   steps:
     cluster_profile: gcp
-    workflow: crc-e2e
+    workflow: code-ready-crc-e2e
```
Comment (cfergeau) [inline ci-operator/config/code-ready/crc/code-ready-crc-master.yaml:38]: I thought the `code-ready` name on github was not great, and that `codeready` would have been preferred?

Comment (gbraad) [reply ci-operator/config/code-ready/crc/code-ready-crc-master.yaml:38]: in general maybe not a 'good name' as it is a brand group

Comment (praveenkumar) [reply ci-operator/config/code-ready/crc/code-ready-crc-master.yaml:38]: @gbraad @cfergeau As of now this is the org name (code-ready) and I don't want to mess around it until we do make some changes github org side.

Comment (cfergeau) [reply ci-operator/config/code-ready/crc/code-ready-crc-master.yaml:38]: After further discussion/tests, the directory hierarchy must match the github hierarchy, so we can't change `code-ready/crc` in paths. The question which remains is whether to fix the non-path naming to use `codeready`, or to be consistent. I agree it's probably less confusing to use `code-ready` for these too at the moment.

**Pull request merged**